src/assets/css/_images.scss

```diff
@@ -366,6 +366,10 @@
 
 .km-icon-quota {
   @include mixins.mask-image('/assets/images/icons/quota.svg', 24px);
+
+  &.i-32 {
```

Comment:
Why do we need this?

`.km-icon-quota` doing same thing this `svg`  exposed as class you can use only `.km-icon-quota` IMO.

Reply:
okay fine